### ISSUE TITLE
Default Enum values should be validated and linked

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/respec#2564]
For example:
```
dictionary SerialOptions {
  ParityType parity = "none";
};

enum ParityType {
  "none",
};
```

We should check if "none" is in `ParityType` and, if it is, we should link to it.

**Comment by marcoscaceres** [comment]
@saschanaz, I guess the default value validation could happen in WebIDL2.js? Might be hard if the Enum is not present in the document... but might be a nice feature.

**Comment by saschanaz** [comment]
Yes, I guess this can be done, given that the enum is in the same document.

**Comment by ridhishjain** [comment]
@marcoscaceres I would like to work on this issue
